pom.xml

```diff
@@ -586,6 +587,12 @@ under the License.
 				<version>${junit4.version}</version>
 			</dependency>
 
+			<dependency>
+				<groupId>org.assertj</groupId>
+				<artifactId>assertj-core</artifactId>
```

Comment:
If the goal is to use assertj everywhere, then we should add it as a general test dependency as we did for junit/hamcrest.

Reply:
Since this also changes the dependency tree, I think we should add a ticket as well.